ITEM_en.tsv

```diff
@@ -4642,62 +4642,62 @@ ITEM_20150428_005028	Increases Movement Speed by +4 for 250 seconds. Right-click
-ITEM_20150428_005055	An instructional manual written a long time ago. It's readable.
-ITEM_20150428_005056	Documents recorded in great king Zachariel. It's readable.
-ITEM_20150428_005057	It looks like warnings from hundreds of years ago. Read it.
-ITEM_20150428_005058	Badly crumpled message. It seems barely readable.
-ITEM_20150428_005059	Ceremony ropes for the wizard Panto
-ITEM_20150428_005060	Fat of Mallardu
-ITEM_20150428_005061	The fat of Mallardu was well saved for burning easily. Let's wake Nepenthes with it.
-ITEM_20150428_005062	The first revelation found in the Crystal Mine. It's readable.
-ITEM_20150428_005063	The soul of Pondel
-ITEM_20150428_005064	The second revelations discovered in the Tenet Chapel. It's readable.
-ITEM_20150428_005065	Meat Loaf of Kepo
-ITEM_20150428_005066	Meat Loaf that can be easily obtained from Kepo. However, the person can never eat.
-ITEM_20150428_005067	The third revelation found in the Thorn Forest. It's readable.
-ITEM_20150428_005068	The fourth revelation found in the Royal Tomb. It's readable.
-ITEM_20150428_005069	The sixth revelation found in the Cathedral. It's readable.
-ITEM_20150428_005070	There are many rumors that can hear the voice of the King at the Royal Tomb Chapel.
-ITEM_20150428_005071	Reagents for testing Graschius's soul.
-ITEM_20150428_005072	Reagents for testing
-ITEM_20150428_005073	It is said to distinguish the monster which killed the pilgrims to the curse of anger to spary the monster. However, this is still a test reagent.
-ITEM_20150428_005074	The fifth revelation found in the Cathedral. It's readable.
+ITEM_20150428_005039	Torn diary page of ab inhabitant who lived in Andale Village. Read what's inside.
+ITEM_20150428_005040	Journal of an unknown adventurer who was exploring the Legwyn family mansion. Readable.
+ITEM_20150428_005041	Reply letters from a man named Beikeol. Readable.
+ITEM_20150428_005042	A statement for someone of the Legwyn family who ordered goods. It's for a medicine that has a dangerous side effect. Readable
+ITEM_20150428_005043	A soiled journal discovered in the Waterway. The diary owner's name is illegible. Readable.
+ITEM_20150428_005044	So old it seems like it will crumble ath the slightest touch. Readable.
+ITEM_20150428_005045	Note left by a pilgrim a very long time ago. Readable.
+ITEM_20150428_005046	It seems like it was left here as a last ditch effort by some pilgrim. Readable.
+ITEM_20150428_005047	A part of a pilgrim's torn diary. Readable.
+ITEM_20150428_005048	A hastily scribble drecord of a pilgrim. Readable.
+ITEM_20150428_005049	Will left by a dying pilgrim. Readable.
+ITEM_20150428_005050	A burnt journal left here by a pilgrim. Readable.
+ITEM_20150428_005051	A lost diary of some kind. Readable.
+ITEM_20150428_005052	A journal that seems to have been written by someone who was angry. Seems readable upon closer examination.
+ITEM_20150428_005053	A man's journal found in a place where mankind is not allowed to enter. Read what's inside.
```

Comment:
A man's journal -> The journal of a person found in a ... 
The Korean string doesn't explicitly say it is a man, but a human person.